src/Blazor.Gitter.Core/content/css/_ChatRoomList.scss

```diff
@@ -1,43 +1,43 @@
-﻿.chat-room-list {
-    padding: 1rem;
-}
-
-.chat-room-list--visible {
-
-}
-
-.chat-room-list__heading {
-    margin-top: 0;
-}
-
-.chat-room-list__list {
-    list-style: none;
-    padding: 0;
-    margin: 0;
-}
-
-.chat-room-list__list-item {
-    padding: 1rem 0;
-}
-
-.chat-room-list__link {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    font-size: $font-size-small;
-    
-    img {
-        width: 35px;
-        margin-right: 1rem;
-        border-radius: .5rem;
-    }
-}
-
-.chat-room-list__unread-items {
-    margin-left: 1rem;
-    width: 33px;
-    text-align: center;
-    color: white;
-    border-radius: 2rem;
-    border: 2px solid #ff6600;
+﻿.chat-room-list {
+    padding: 1rem;
+}
+
+.chat-room-list--visible {
+
+}
+
+.chat-room-list__heading {
+    margin-top: 0;
+}
+
+.chat-room-list__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.chat-room-list__list-item {
+    padding: 1rem 0;
+}
+
+.chat-room-list__link {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-size: $font-size-small;
+    
+    img {
+        width: 35px;
+        margin-right: 1rem;
+        border-radius: .5rem;
+    }
+}
+
+.chat-room-list__unread-items {
+    margin-left: 1rem;
+    width: 33px;
+    text-align: center;
+    color: white;
+    border-radius: 2rem;
+    border: 2px solid #ff6600;
 }
```

Comment:
What changed here? It looks like nothing did - am I missing something?

Reply:
I didn't touch the file (other than moving it). Is there a whitespace difference?

Reply:
Probably - that's fine then